gfxdraw unit tests test_aapolygon and test_pie fail

* OS: Debian Gnu/Linux, and Mac.
* hardware: 32 bit Intel, and 64bit.
* Compiler: gcc 4.4.5 for Pygame and Python interpreter

The gfxdraw unit tests return the following fail messages.

```
python test/gfxdraw_test.py
```
    
    ======================================================================
    FAIL: GfxdrawDefaultTest.test_aapolygon
    ----------------------------------------------------------------------
    Traceback (most recent call last):
      File "/home/lenard/.local/lib/python3.2/site-packages/pygame/tests/gfxdraw_test.py", line 613, in test_aapolygon
        self.check_not_at(surf, posn, bg_adjusted)
      File "/home/lenard/.local/lib/python3.2/site-packages/pygame/tests/gfxdraw_test.py", line 72, in check_not_at
        self.failIfEqual(sc, color, fail_msg)
    AssertionError: (255, 255, 255, 255) != (255, 255, 255, 255) at (11, 80), bitsize: 8, flags: 0, masks: (96, 28, 3, 0)
    
    ======================================================================
    FAIL: GfxdrawDefaultTest.test_pie
    ----------------------------------------------------------------------
    Traceback (most recent call last):
      File "/home/lenard/.local/lib/python3.2/site-packages/pygame/tests/gfxdraw_test.py", line 477, in test_pie
        self.check_at(surf, posn, bg_adjusted)
      File "/home/lenard/.local/lib/python3.2/site-packages/pygame/tests/gfxdraw_test.py", line 65, in check_at
        self.failUnlessEqual(sc, color, fail_msg)
    AssertionError: (128, 64, 8, 255) != (255, 255, 255, 255) at (45, 70), bitsize: 8, flags: 0, masks: (96, 28, 3, 0)
        
Originally reported by: **Lenard Lindstrom (Bitbucket: [llindstrom](https://bitbucket.org/llindstrom), GitHub: [llindstrom](https://github.com/llindstrom))**

----------------------------------------
- Bitbucket: https://bitbucket.org/pygame/pygame/issue/80

--- a/test/gfxdraw_test.py
+++ b/test/gfxdraw_test.py
@@ -1,5 +1,5 @@
 # Two unit tests fail! Disable to allow automated builds to continue.
-__tags__ = ('ignore', 'subprocess_ignore')
+#__tags__ = ('ignore', 'subprocess_ignore')
 
 import unittest
 import pygame
@@ -572,12 +572,7 @@ class GfxdrawDefaultTest( unittest.TestCase ):
         fg = self.foreground_color
         bg = self.background_color
         points = [(10, 80), (10, 15), (92, 25), (92, 80)]
-        fg_test_points = (points +
-                          [(points[0][0], points[0][1] - 1),
-                           (points[0][0] + 1, points[0][1]),
-                           (points[3][0] - 1, points[3][1]),
-                           (points[3][0], points[3][1] - 1),
-                           (points[2][0], points[2][1] + 1)])
+        fg_test_points = points
         bg_test_points = [(points[0][0] - 1, points[0][1]),
                           (points[0][0], points[0][1] + 1),
                           (points[0][0] - 1, points[0][1] + 1),
@@ -596,7 +591,9 @@ class GfxdrawDefaultTest( unittest.TestCase ):
             bg_adjusted = surf.unmap_rgb(surf.map_rgb(bg))
             pygame.gfxdraw.aapolygon(surf, points, fg)
             for posn in fg_test_points:
-                self.check_not_at(surf, posn, bg_adjusted)
+                self.check_at(surf, posn, fg_adjusted)
+            for posn in bg_test_points:
+                self.check_not_at(surf, posn, fg_adjusted)
             for posn in bg_test_points:
                 self.check_at(surf, posn, bg_adjusted)
 

--- a/test/gfxdraw_test.py
+++ b/test/gfxdraw_test.py
@@ -1,6 +1,3 @@
-# Two unit tests fail! Disable to allow automated builds to continue.
-#__tags__ = ('ignore', 'subprocess_ignore')
-
 import unittest
 import pygame
 import pygame.gfxdraw
@@ -452,7 +449,7 @@ class GfxdrawDefaultTest( unittest.TestCase ):
                           (x + 1, y + 1),
                           (x + r + 1, y),
                           (x + r, y - 1),
-                          (x, y + r)]
+                          (x, y + r + 1)]
         for surf in self.surfaces:
             fg_adjusted = surf.unmap_rgb(surf.map_rgb(fg))
             bg_adjusted = surf.unmap_rgb(surf.map_rgb(bg))

